<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1100</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Carmelita Engine</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="Hierarchywidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>200</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Hierarchy</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QDockWidget" name="Inspectorwidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>450</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_7"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1100</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRenderer">
    <property name="title">
     <string>Renderer</string>
    </property>
    <addaction name="actionAlbedo"/>
    <addaction name="actionLighting"/>
    <addaction name="actionNormals"/>
    <addaction name="actionDepth_test"/>
    <addaction name="separator"/>
    <addaction name="actionBlur"/>
    <addaction name="actionDepth_of_Field"/>
   </widget>
   <widget class="QMenu" name="menuLight_Settings">
    <property name="title">
     <string>Light</string>
    </property>
    <addaction name="actionLight_Settings"/>
    <addaction name="actionLightColor"/>
   </widget>
   <widget class="QMenu" name="menuDoF_Options">
    <property name="title">
     <string>DoF Options</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <addaction name="menuRenderer"/>
   <addaction name="menuLight_Settings"/>
   <addaction name="menuDoF_Options"/>
  </widget>
  <widget class="QDockWidget" name="Scene">
   <property name="minimumSize">
    <size>
     <width>777</width>
     <height>701</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Scene</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <action name="actionOpen_Project">
   <property name="text">
    <string>Open Project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="text">
    <string>Save Project</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionAlbedo">
   <property name="text">
    <string>Albedo</string>
   </property>
  </action>
  <action name="actionLighting">
   <property name="text">
    <string>Lighting</string>
   </property>
  </action>
  <action name="actionNormals">
   <property name="text">
    <string>Normals</string>
   </property>
  </action>
  <action name="actionDepth_test">
   <property name="text">
    <string>Depth test</string>
   </property>
  </action>
  <action name="actionLight_Settings">
   <property name="text">
    <string>Light Direction</string>
   </property>
  </action>
  <action name="actionLightColor">
   <property name="text">
    <string>Light Color</string>
   </property>
  </action>
  <action name="actionBlur">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Blur</string>
   </property>
  </action>
  <action name="actionDepth_of_Field">
   <property name="text">
    <string>Depth of Field</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>Options</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
